torch/csrc/jit/runtime/symbolic_script.cpp

```diff
@@ -612,7 +612,7 @@ const std::vector<std::string> functions = {
             size = len(tensors)
             split_sizes = [0] * size
             for i in range(size):
-                if tensors[i].numel() > 0:
+                if tensors[i].numel() > 0 or len(tensors[i].size()) != 1:
```

Comment:
nit: `len(tensors[i].size()) != 1` -> `tensors[i].dim() != 1`

Reply:
heh, I tried `.ndim` and `.ndimension()`, and neither compiled with script, I guess `.dim()`'s the ticket. But you are right above, these 2 conditions are equivalent to `tensors[i].size() != [0]`

Reply:
`.size() != [0]` should work, .size() returns a list of ints in TS..

Reply:
Ah interesting, it doesn't work in eager
```
In [8]: a.size() == [0]
Out[8]: False

In [9]: a.size() == torch.Size([0])
Out[9]: True
```